app/src/main/java/com/example/music/data/source/local/SongAsynTask.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.music.data.source.local
+
+import android.os.AsyncTask
+import com.example.music.data.entity.Song
+import com.example.music.data.source.OnSongLoadedCallback
+import java.lang.Exception
+
+class SongAsynTask(
+    private val songHandler: SongHandler,
+    private val callback: OnSongLoadedCallback
+) : AsyncTask<Void, Void, MutableList<Song>>() {
+    private var exception: Exception? = null
```

Comment:
để 1 dòng trống dưới header 
```suggestion
) : AsyncTask<Void, Void, MutableList<Song>>() {

    private var exception: Exception? = null
```